hasher-matcher-actioner/webapp/src/components/settings/ActionRulesTableRow.tsx

```diff
@@ -94,19 +94,24 @@ export default function ActionRulesTableRow({
           case 'Classification':
             ret += ' MatchedContent';
             break;
+          case 'SubmittedContent':
+            ret += ' SubmittedContent';
+            break;
           default:
             ret += ` ${classification.classificationType}`;
             break;
         }
 
         if (classification.equalTo) {
           ret +=
-            classification.classificationType === 'Classification'
+            classification.classificationType === 'Classification' ||
+            classification.classificationType === 'SubmittedContent'
```

Comment:
Could you include a screenshot of this change if it is visbal on the setting UI? I have the same nit about naming as well. This one is worse imo because it is a Classification just a different kind right?

Reply:
Added screenshot.

Discussed offline